srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/DiscreteNumericalValue.java

```diff
@@ -13,6 +13,14 @@
     private final Category fSelectedCategory;
     private final float fNumericalValue;
     
+    /**
+     * Business code should call {@link #fromCategory(DiscreteNumericalVariable, Category)}
```

Comment:
"Business code should call...": not exactly true. This constructor is protected precisely to allow business code (in subclasses) to call it, though none does yet. Perhaps this class should be final, though, because I didn't really think through how subclasses would extend it so it's probably half-baked.